src/templates/rust-all/Makefile.tftpl

```diff
@@ -24,6 +24,7 @@ include .make/docker.mk
 release: rust-release
 test: rust-test-all cspell-test toml-test python-test md-test-links editorconfig-test
 tidy: rust-tidy toml-tidy python-tidy
+format-all: rust-tidy toml-tidy python-tidy editorconfig-tidy
```

Comment:
I think you can just add the `editorconfig-tidy` target to the combined `tidy` target so we don't need a separate `format-all` target for this.

Reply:
ok, I thought as editorconfig-tidy can theoretically produce breaking changes, we might want to have a separate target, but the probability of breaking changes is low so I guess we can just add it there.

Reply:
Everything in the 'tidy' target can introduce breaking changes. That's why we don't want it to be part of the pre-commit hook, it should be intentionally run by the user.